packages/mysql.rb

```diff
@@ -0,0 +1,74 @@
+require 'package'
+
+class Mysql < Package
+  description "MySQL Community Edition is a freely downloadable version of the world's most popular open source database"
+  homepage 'https://www.mysql.com/'
+  version '8.0.24'
+  license 'GPL-2'
+  compatibility 'x86_64'
+  source_url 'https://dev.mysql.com/get/Downloads/MySQL-8.0/mysql-8.0.24-linux-glibc2.17-x86_64-minimal.tar.xz'
+  source_sha256 'ea7e67582b2a6816f9d74c162416cf3b97b539d39aef77e3e251e834ce6c06d5'
+
+  binary_url ({
+  })
+  binary_sha256 ({
+  })
+
+  depends_on 'numactl'
+
+  def self.install
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/bin"
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/mysql"
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/share"
+    FileUtils.mv 'man/', "#{CREW_DEST_PREFIX}/share"
+    FileUtils.mv Dir.glob('*'), "#{CREW_DEST_PREFIX}/mysql"
+    Dir.chdir "#{CREW_DEST_PREFIX}/mysql/bin" do
```

Comment:
You can rubyize it with:
```ruby
Dir["#{CREW_DEST_PREFIX}/mysql/bin/*"].map {|f| f.sub("#{CREW_DEST_DIR}", '').each do |f|
  FileUtils.ln_s f, "#{CREW_DEST_PREFIX}/bin/"
end
```

Reply:
I get the following error:
```
SyntaxError: /usr/local/lib/crew/packages/mysql.rb:33: syntax error, unexpected `end', expecting '}'
  end
  ^~~
/usr/local/lib/crew/packages/mysql.rb:77: syntax error, unexpected end-of-input, expecting `end'
```
I'm in agreement with the spirit of your suggestion, however.